rpcd/playbooks/group_vars/all.yml

```diff
@@ -15,8 +15,8 @@
 # This is the location where the rpc repository was checked out
 rpc_repo_path: /opt/rpc-openstack/openstack-ansible
 
-# rpc-openstack version
-rpc_release: r11.1.0rc5
+# rpc-openstack version, this should be discovered by git_repo_info
+rpc_release: "{{ rpc_openstack_repo is defined | ternary((rpc_openstack_repo | default({}))['version'], 'UNKNOWN_RELEASE') }}"
```

Comment:
I'm a little confused how the value here actually gets set. There's no reference to the rpc_openstack_repo value outside of this file, so it's not registered in the Ansible discover roles, nor in the Python module.

Am I missing a link here?